### PR TITLE
Calvin/requests

### PR DESCRIPTION
**Pull request by calvind777**
## Feature Name
Feature Description

### Screenshots

CC: @clairelin135

### CODE REVIEW

**Comment by calvind777** [comment]
created API controllers for artists and works, created initial view of all of artists works (see '/artists/0'), fixed requests.jsx

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
   devise_for :artists
   devise_for :buyers
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root 'pages#home'
+
+
+
+  namespace :api, defaults: { format: :json } do
+    resources :works, :only => [:show, :create, :update, :destroy]
+    resources :artists, :only => [:show, :create, :update, :destroy]
```
Comment (clairelin135) [inline config/routes.rb]: Remove create/update/destroy functions

File: app/controllers/api/artists_controller.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  end
+
+  def update
+    artist = Artist.find(params[:id])
+    new_work = work.update(params)
+    render_json_message(:ok, message: 'Request successfully updated!')
+  end
+
+  def destroy
+    artist = Artist.find(params[:id])
```
Comment (clairelin135) [inline app/controllers/api/artists_controller.rb]: Remove functions

File: app/javascript/components/artist_works.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    super(props);
+    this.state = {
+      works: []
+    }
+  }
+
+  componentDidMount = () => {
+    const artist_id = this.props.artist.id;
+    Requester.get('/api/artists/works/'+artist_id, (response) => {
+      this.setState({works: response});
```
Comment (clairelin135) [inline app/javascript/components/artist_works.js]: Spaces in between curly braces

File: app/javascript/components/artist_works.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+      console.log(response);
```
Comment (clairelin135) [inline app/javascript/components/artist_works.js]: Remove print

File: app/javascript/components/artist_works.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    }, (response) => {console.log(response)});
```
Comment (clairelin135) [inline app/javascript/components/artist_works.js]: console.error

File: app/javascript/components/artist_works.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      works: []
+    }
+  }
+
+  componentDidMount = () => {
+    const artist_id = this.props.artist.id;
+    Requester.get('/api/artists/works/'+artist_id, (response) => {
```
Comment (clairelin135) [inline app/javascript/components/artist_works.js]: store in APIRoutes

**Pull request merged**